bindings/engine.i

```diff
@@ -78,7 +78,7 @@ GLIST_HELPER_INOUT(CommodityList, SWIGTYPE_p_gnc_commodity);
     SCM lst = SCM_EOL;
     for (GList* node = $1; node; node = node->next)
     {
-        int *date = node->data;
+        time64 *date = node->data;
         lst = scm_cons (scm_from_int64 ((int)*date), lst);
```

Comment:
That's a problem: Casting it to an int will chop off the high 32 bits on systems where int is int32_t. The guile headers have
`SCM_API SCM          scm_from_int64  (scm_t_int64 x);`
and
`typedef int64_t scm_t_int64;`
Since time64 is also a typedef of int64_t there should be no need for a cast. What error are you getting without the cast? What happens if you cast to `scm_t_int64` instead?

Reply:
> Since time64 is also a typedef of int64_t there should be no need for a cast. What error are you getting without the cast?

No error :)